README.md

```diff
@@ -1,24 +1,47 @@
-# README
+# VocabiApp
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## What is this app?
 
-Things you may want to cover:
+This a **vocabulary trainer for foreign languages**, the game is as follows,
+you sign up (just using you email), specify your **original** language and your
+**target** language (the one on that you want to improve your vocabulary)
+pick a *part of speech* category, i.e, pronouns, verbs, etc. After selecting
+the category, click on a level, inside the level you'll need to translate the
+displayed words into your original language, answering correctly gives you
+points! The user with more points will appear at the top of the ranking list of
+the home page. Pretty straightforward, isn't it? Well, that kind of explains
+everything about the app, have a good time while using it! And, if you want to,
+collaborate to its development!
```

Comment:
Nice intro, very clear and concise. 
I've pasted the text into Grammarly, it finds a couple of fixes. 


![image](https://user-images.githubusercontent.com/9512604/70051376-adf20180-15af-11ea-8103-7bdcf8d7e8ca.png)
1. you -> your
2. level -> level,
3. to -> with

Reply:
Oh! The next time I'll remember to pass the paragraphs through **grammarly**.